Add one line description for ARX (#947)
opendp/dp-wizard: README.md

From other groups:

- [PrivSyn](https://github.com/vvv214/privsyn-tabular): Uses AIM for synthetic data generation.

> - [ARX](https://arx.deidentifier.org/): Differential privacy [is mentioned](https://arx.deidentifier.org/overview/privacy-criteria/), but only as one model among many.

Author: @eddiestudies , is this what you'd prefer?
```suggestion
- [ARX](https://arx.deidentifier.org/): Differential privacy [is mentioned](https://arx.deidentifier.org/overview/privacy-criteria/) as one of the supported models.
```
